### PR TITLE
#160168277 Get profile information from travel request page

### PR DESCRIPTION
**Pull request by Collins33**
#### What does this PR do?
Get personal information from the new travel request page.

#### Description of Task to be completed?
When a logged in user makes a travel request, the following happens:
- They fill the form with their personal details and travel details.
- If they select the checkbox, the personal user details are saved and when they make another request, the personal details fields are prepopulated.
- If they do not select the checkbox, the personal details are not saved and when he makes another travel request, the fields are not prepopulated.

#### How should this be manually tested?
- Git clone `https://github.com/andela/travel_tool_front.git`
- cd into `travel_tool_front`
- Run `make ssh` on the terminal
- Then run `yarn install`
- Run the server with `make start`
- To be able to log in, add a supported domain name to your hosts:
  - `vim /etc/hosts` and add the line `127.0.0.1        local.andela.com`
  - Create a `.env` file and add the following : 
     -  REACT_APP_ANDELA_AUTH_HOST=http://api-staging.andela.com
     -  REACT_APP_AUTH_REDIRECT_URL=http://travel-tool.andela.com:3000
- Click the login button which will redirect you to the requests page after authentication.
- Click the new request button and submit a travel request with the checkbox clicked and when checkbox is not clicked
#### Any background context you want to provide?
None

#### What are the relevant pivotal tracker stories?
#160168277

#### Screenshots (if appropriate)
screen shot 2018-09-03 at 17 09 18

#### Questions:
None

### CODE REVIEW

File: src/components/CheckBox/index.jsx [hunk 1]
```diff
@@ -0,0 +1,62 @@
+import React, { PureComponent } from 'react';
+import PropTypes from 'prop-types';
+import './Checkbox.scss';
+import tick from '../../images/Tick/tick.svg';
+
+class Checkbox extends PureComponent{
+  state = {
+    checkBoxClickedClicked: 'notClicked'
```
Comment (hariclerry) [inline src/components/CheckBox/index.jsx]: Is `checkBoxClickedClicked` a typo or is it suppose to be like that?

File: src/components/CheckBox/index.jsx [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+      <div className={className} onClick={()=>{this.clickCheckbox();}} role="button" tabIndex="0" onKeyDown={()=>{this.clickCheckbox();}} id="checkBox">
+        <img src={tick} alt="clicked" className="ticksvg" />
+      </div>
+    );
+  }
+
+  renderText=()=>{
+    return(
+      <p>
+        Remember this details
```
Comment (johnmutuma5) [inline src/components/CheckBox/index.jsx]: Use the plural pronoun `these`

**Comment by samuelkubai** [comment]
@Collins33 this should be updated to the correct label.

**Comment by samuelkubai** [comment]
@daddychukz I have merged it.

@Collins33 please resolve the conflicts.

**Comment by daddychukz** [comment]
LGTM @samuelkubai

**Pull request merged**